src/supertux/player_status_hud.cpp

```diff
@@ -85,6 +87,49 @@ PlayerStatusHUD::draw(DrawingContext& context)
                             LAYER_HUD,
                             PlayerStatusHUD::text_color);
 
+  std::string ammo_text;
+
+  if (m_player_status.bonus == FIRE_BONUS) {
+
+    ammo_text = std::to_string(m_player_status.max_fire_bullets);
+
+    if (fire_surface) {
+      context.color().draw_surface(fire_surface,
+                                   Vector(static_cast<float>(context.get_width()) - BORDER_X - static_cast<float>(fire_surface->get_width()) - Resources::fixed_font->get_text_width(ammo_text),
+                                          BORDER_Y + 1.0f + (Resources::fixed_font->get_text_height(coins_text) + 5) + (Resources::fixed_font->get_text_height(ammo_text) + 5) * static_cast<float>(player_id)),
+                                   LAYER_HUD);
```

Comment:
Please insert line breaks if the text exceeds 80 characters. 
If the text goes too far to the right, it is difficult to read on github (bad example [here](http://github.com/SuperTux/supertux/blob/02631a48ccaf9a3924ea4b9f7a0f8886ec38ff98/src/supertux/player_status_hud.cpp#L75)), or when using a text editor with small width.
See also: https://01.org/linuxgraphics/gfx-docs/drm/process/coding-style.html